### ISSUE TITLE
[BUG] Parcel v2 namer bundle group conflict on development server

### ISSUE THREAD

**Issue by ghost** [101arrowz/pwa-manifest#28]
## Describe the bug
using Parcel v2 development server with the config will result in broken build process.

## Expected behavior
work normally with `serve` command

## Actual behavior
using config with `serve` command will result in first time successful build but after file change and attempt to rebuild, Parcel will complain about more than one bundler on `namer` group.

## To reproduce
add `parcel-config-pwa-manifest` to a Parcel v2 environment and run `parcel serve index.html`

## Additional context
building _React_ for _browser_.

`package.json` :
```json
{
    "pwaManifest": {
        "name": "test",
        "shortName": "test",
        "startURL": "./offline",
        "theme": "#000000",
        "generateIconOptions": {
            "baseIcon": "./static/logoface.svg",
            "sizes": [
                32,
                64,
                96,
                152,
                192,
                384,
                512
            ],
            "genFavicons": true
        }
    }
}
```
`.parcelrc` :
```json
{
    "extends": [
        "@parcel/config-default",
        "parcel-config-pwa-manifest"
    ],
    "transformers": {
        "*.svg": [
            "@parcel/transformer-raw"
        ],
        "*.png": [
            "@parcel/transformer-raw"
        ]
    }
}
````

`.babelrc` : 
```json
{
    "presets": ["@parcel/babel-preset-env", "@babel/preset-react"],
    "plugins": [
        "@babel/plugin-syntax-dynamic-import",
        "@babel/plugin-proposal-class-properties",
        "@parcel/babel-plugin-transform-runtime"
    ]
}
```

logs:
```
🚨 Build failed.
@parcel/namer-default: Bundle group cannot have more than one entry bundle of the same type
AssertionError [ERR_ASSERTION]: Bundle group cannot have more than one entry bundle of the same type
    at Object.name (/home/user/test/node_modules/@parcel/namer-default/lib/DefaultNamer.js:85:29)
    at BundlerRunner.nameBundle (/home/user/test/node_modules/@parcel/core/lib/BundlerRunner.js:378:39)
    at async Promise.all (index 39)
    at async BundlerRunner.nameBundles (/home/user/test/node_modules/@parcel/core/lib/BundlerRunner.js:365:5)
    at async BundlerRunner.bundle (/home/user/test/node_modules/@parcel/core/lib/BundlerRunner.js:324:5)
    at async Parcel._build (/home/user/test/node_modules/@parcel/core/lib/Parcel.js:668:11)
    at async Parcel._startNextBuild (/home/user/test/node_modules/@parcel/core/lib/Parcel.js:546:24)
    at async PromiseQueue._runFn (/home/user/test/node_modules/@parcel/utils/lib/PromiseQueue.js:108:7)
    at async PromiseQueue._next (/home/user/test/node_modules/@parcel/utils/lib/PromiseQueue.js:95:5)
```

## Temporary Workaround
just disable the config on `development` environment:

`package.json`  : 
```json
{
    "pwaManifest": {
        "development":{
            "disabled": true
        }
    }
}
```

**Comment by 101arrowz** [comment]
I've been getting this too. I would disable the plugin in dev anyway, bug I'll look into the issue.

**Comment by GauBen** [comment]
Hello, this bug also happens when running `parcel build`.

Here is a reproducible run: https://github.com/GauBen/gauben.github.io/runs/2554449658

**Comment by 101arrowz** [comment]
Fixed in the next release, check #40 for updates. Thanks for the report!
